pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -31,6 +31,6 @@ stdenv.mkDerivation {
     homepage = "https://github.com/rockchip-linux/rkbin";
     license = licenses.unfreeRedistributable;
     maintainers = with maintainers; [ thefossguy ];
-    platforms = [ "aarch64-linux" ];
+    platforms = lib.platforms.all;
```

Comment:
`lib.platform.linux` maybe. `box` ony runs on those.

Reply:
@flokli so shall I set `platforms = box64.meta.platforms;` ?

Reply:
Well, that's an implementation detail. Just `lib.platforms.linux` should be fine, the rest doesn't run on hydra anyways :shrug:

Reply:
Ok, after some discussion, all might be correct. Merging.

Reply:
rkbin is lib.platforms.all because it is just moving files, no builds are happening, whereas rkboot is aarch64/x86 only, because it involves building.